ts/examples/web-examples/mp4/src/mp4.ts

```diff
@@ -104,22 +104,24 @@ export async function play(videoUrl) {
       const frameOptions = {
         format: 'RGBA',
       };
-      const buffer = new Uint8ClampedArray(frame.allocationSize(frameOptions));
-      frame.copyTo(buffer, frameOptions);
+      const buffer = new Uint8ClampedArray(
+        frame.allocationSize(frameOptions as VideoFrameCopyToOptions)
```

Comment:
`as VideoFrameCopyToOptions` is needed here because the default typings for `VideoFrame` are missing `format` field, even though `format` is part of the web codecs spec